crypto/keyring.go

```diff
@@ -296,6 +296,11 @@ func (keyRing *KeyRing) ReadFromJSON(jsonData []byte) (err error) {
 	return keyRing.newKeyRingFromPGPKeyObject(keyObjs)
 }
 
+// UnmarshalJSON is an alias for ReadFromJSON to implement the encoding/json.Unmarshaler interface
+func (keyRing *KeyRing) UnmarshalJSON(jsonData []byte) (err error) {
+	return keyRing.ReadFromJSON(jsonData)
+}
```

Comment:
This is a bit nitpicky but - I don't think we should really have two public APIs for this. It sort of seems to imply that this is an important / good way to serialize / deserialize a list of keys, which I'm not so sure about. It seems better to have a list of armored keys or multiple keys armored together. So I know golang's names are weird but I'd just revert to having `UnmarshalJSON` for compatibility and nothing more.